app/models/article.rb

```diff
@@ -200,6 +200,10 @@ def mark_as_deleted
     update_column :deleted_at, Time.now
   end
 
+  def current_article_price
+    article_prices.find_or_initialize_by(price: price, tax: tax, deposit: deposit, unit_quantity: unit_quantity)
+  end
+
```

Comment:
Does this also get the _latest_ `ArticlePrice` with these values? Since the `article_prices` are as much as a price history of the article (as far as I understand it), it's good to point to the right object so that an `order_article` gets the correct record on order close.